lib/omnibus/packagers/windows_base.rb

```diff
@@ -149,6 +152,7 @@ def try_sign(package_file, url)
         arr << "/d #{project.package_name}"
         arr << "\"#{package_file}\""
       end.join(" ")
+      puts cmd
```

Comment:
Is any  of the cmd info sensitive or is it safe to print it ?

Reply:
there's nothing secret in there.